utils/polus-imagej-util/{{cookiecutter.project_slug}}/src/ij_converter.py

```diff
@@ -0,0 +1,250 @@
+
+'''
+A conversion utility built to convert abstract to primitive
+This works for Threshold Apply and Gaussian Filter functions
+Note change from jnius to jpype for handling conversions in ij_converter.py
+
+'''
+
+import imagej
+import logging
+import imglyb
+import jpype
+import numpy as np
+import scyjava
+
+# Initialize the logger
+logging.basicConfig(format='%(asctime)s - %(name)-8s - %(levelname)-8s - %(message)s',
+                    datefmt='%d-%b-%y %H:%M:%S')
+logger = logging.getLogger("ij_converter")
+logger.setLevel(logging.INFO)
+
+ij = None
```

Comment:
Per #144 , we should get away from using this as a global variable.

Reply:
I solved this by passing the imagej instance `ij` directly to the `ij_converter,py` functions.